source/examples/queue-parallel.cpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2020 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <CL/sycl.hpp>
+#include <iostream>
+
+const int count = 10;
+
+int main() {
+
+  // Data initialized to zeros.
+  int data[count] = {0};
+
+  auto q = sycl::queue(sycl::default_selector());
+  {
+    sycl::buffer<int, 1> data_buffer(data, sycl::range<1>(count));
+
+    q.submit([&](sycl::handler &cgh) {
+      auto data_accessor =
+          data_buffer.get_access<sycl::access::mode::read_write>(cgh);
+      cgh.parallel_for<class add_five>(sycl::range<1>(count),
+                                       [=](sycl::item<1> item) {
+                                         auto index = item.get_id(0);
+                                         data_accessor[index] += 5;
+                                       });
+    });
+    q.wait();
+  }
+
+  for (int i = 0; i < count; ++i) {
+    std::cout << data[i] << " ";
+  }
```

Comment:
There is a lacking end-of-line.
You could also use a for-range-loop like:
```suggestion
  for (auto e : data)
    std::cout << data[i] << " ";
  std::cout << std::endl;
```